README.md

```diff
@@ -1,43 +1,46 @@
-**Prophesee FPGA Projects**
-===========================
+# Prophesee FPGA Projects
 
-Overview
---------
+## Overview
 
-This Prophesee repository contains the source files and scripts necessary to build Prophesee FPGA projects.
+This repository contains the source files and scripts necessary to build Prophesee FPGA projects.
 
-For additional information or support, please contact Prophesee Support at [support@prophesee.ai](mailto:support@prophesee.ai)
+## Requirements
 
-Package Structure
------------------
+The projects in this repository have been tested and validated on the following setup:
 
-### Contents
+- Ubuntu 20.04.6 LTS
+- AMD Vivado 2022.2 (64-bit) with installed support for Zynq US+ Devices
```

Comment:
maybe add more details on the minimal set of options that should be selected during Vivado install